content/the-ember-times-issue-176.md

```diff
@@ -31,15 +33,13 @@ Here at the Ember Times, we are super excited about the conference [lineup](http
 
 ---
 
-## [2. Section title in sentence case 🐹](section-url)
+## [The ins and outs of organising EmberConf 📻](https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+In the latest [Ember Report](https://twitter.com/EmberReport) [podcast]((https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021)) [Leah Silber (@wifelette)](https://twitter.com/wifelette) talks about organising EmberConf remote, the history of EmberConf and the topics like [**inclusion**](https://emberconf.com/about/inclusiveness-at-emberconf) that are so crucial to the EmberConf vibe.
```

Comment:
I noticed that the link for `podcast` didn't show up. I made a few other suggestions that I think capture the meaning that you had. Can you have a look?

```suggestion
In the latest [Ember Report](https://twitter.com/EmberReport) [podcast](https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021), [Leah Silber (@wifelette)](https://twitter.com/wifelette) talks about organising a remote EmberConf, the history of EmberConf, and additional topics like [inclusiveness](https://emberconf.com/about/inclusiveness-at-emberconf) that is so crucial to the EmberConf vibe.
```

Reply:
Looks good, thank you both! I'm hoping to submit a bit earlier today than normal, so am going to merge in these changes while have some cycles.